examples/epstein_civil_violence/epstein_civil_violence/model.py

```diff
@@ -60,10 +60,21 @@ def __init__(
         self.iteration = 0
         self.schedule = mesa.time.RandomActivation(self)
         self.grid = mesa.space.SingleGrid(width, height, torus=True)
+
+        # agent counts
+        self.citizen_count = 0
```

Comment:
Why are those numbers not collected via the data collector framework?

Reply:
I couldn't get the graph to update off of data collector values, so this was the method I could get to display the graph update on each step for Citizen agent state counts in the web interface.

Reply:
I didn't see these attributes being used in any plots.

Reply:
@JoeHelbing any update?

Reply:
Aye, sorry. Made most of these changes a couple months ago for the class so had to reacquaint myself with what I was doing there. 

Agent count attributes were part of a set of TextElement visualization modules in server.py I was using to track and confirm model behavior that I removed after testing. Looks like I left the attributes. Removing these as they are unneeded.